docs/source/execution_review.md

```diff
@@ -0,0 +1,142 @@
+
+This extracted tar file becomes the working directory for your job. As an example, given the following directory structure with `subpath="src"`:
+```
+- docs
+- src
+  - your_library
+- tests
+```
+Your working directory at the time of execution will look like:
+```
+- your_library
+```
+If you're executing a Python function, this working directory will automatically be included in your Python path.
+
+> **_NOTE:_** git archive doesn't package uncommitted changes. In the future, we may add support for including uncommitted changes while honoring `.gitignore`.
+
+### Defining Executors
+Next, We'll describe details on setting up each of the executors below.
+
+#### LocalExecutor
+
+The LocalExecutor is the simplest executor. It executes your task locally in a separate process or group from your current working directory.
+
+The easiest way to define one is to call `run.LocalExecutor()`.
+
+#### SlurmExecutor
+
+The SlurmExecutor enables launching the configured task on a Slurm Cluster with Pyxis.  Additionally, you can configure a `run.SSHTunnel`, which enables you to execute tasks on the Slurm cluster from your local machine while NeMo-Run manages the SSH connection for you. This setup supports use cases such as launching the same task on multiple Slurm clusters.
+
+Below is an example of configuring a Slurm Executor
+```python
+def your_slurm_executor(nodes: int = 1, container_image: str = DEFAULT_IMAGE):
+    # SSH Tunnel
+    ssh_tunnel = run.SSHTunnel(
+        host="your-slurm-host",
+        user="your-user",
+        job_dir="directory-to-store-runs-on-the-slurm-cluster",
+        identity="optional-path-to-your-key-for-auth",
+    )
+    # Local Tunnel to use if you're already on the cluster
+    local_tunnel = run.LocalTunnel()
+
+    packager = GitArchivePackager(
+        # This will also be the working directory in your task.
+        # If empty, the working directory will be toplevel of your git repo
+        subpath="optional-subpath-from-toplevel-of-your-git-repo"
+    )
+
+    executor = run.SlurmExecutor(
+        # Most of these parameters are specific to slurm
+        account="your-account",
+        partition="your-partition",
+        ntasks_per_node=8,
+        gpus_per_node=8,
+        nodes=nodes,
+        tunnel=ssh_tunnel,
+        container_image=container_image,
+        time="00:30:00",
+        env_vars=common_envs(),
+        container_mounts=mounts_for_your_hubs(),
+        packager=packager,
+    )
+
+# You can then call the executor in your script like
+executor = your_slurm_cluster(nodes=8, container_image="your-nemo-image")
+```
+
+Use the SSH Tunnel when launching from your local machine, or the Local Tunnel if you’re already on the Slurm cluster.
+
+#### SkypilotExecutor
+This executor is used to configure [Skypilot](https://skypilot.readthedocs.io/en/latest/docs/index.html). Make sure Skypilot is installed and atleast one cloud is configured using `sky check`.
```

Comment:
fix spacing.

This executor is used to configure [Skypilot](https://skypilot.readthedocs.io/en/latest/docs/index.html). Make sure Skypilot is installed and at least one cloud is configured using `sky check`.